cpplite/cpplite.debugger/src/org/netbeans/modules/cpplite/debugger/CPPLiteDebuggerConfig.java

```diff
@@ -31,16 +32,30 @@
 public final class CPPLiteDebuggerConfig {
 
     private final List<String> executable;
-    private final File directory;
+    private final ExplicitProcessParameters processParameters;
     @NullAllowed
     private final Long processId;
     private final String debugger;
 
-    public CPPLiteDebuggerConfig(List<String> executable, File directory, @NullAllowed Long processId, String debugger) {
-        this.executable = executable;
-        this.directory = directory;
+    public CPPLiteDebuggerConfig(List<String> executable, ExplicitProcessParameters processParameters, @NullAllowed Long processId, String debugger) {
+        this.processParameters = processParameters;
         this.processId = processId;
         this.debugger = debugger;
+        if (processParameters.isArgReplacement()) {
```

Comment:
I see that `launcherArguments` property is ignored - would it be possible to e.g. log a warning, if someone passes them (since they won't be used) ? Doing so may indicate some bug.

Reply:
Right. An IllegalStateException is logged.